en_ewt-ud-test.conllu

```diff
@@ -23443,7 +23443,7 @@
 
 # sent_id = answers-20111106103415AAqdokn_ans-0005
 # text = Fudruckers.
-1	Fudruckers	Fuddruckers	PROPN	NNPS	Number=Sing|Typo=Yes	0	root	0:root	CorrectForm=Fuddruckers|SpaceAfter=No
+1	Fudruckers	Fuddrucker	PROPN	NNPS	Number=Plur|Typo=Yes	0	root	0:root	CorrectForm=Fuddruckers|SpaceAfter=No
```

Comment:
Should be `NNP` (company name ends in "s")

Reply:
Thanks. I've fixed this during the merge.

Reply:
Fixed. Thanks.